src/module/ovs/bpf.rs

```diff
@@ -122,6 +130,10 @@ pub(super) fn unmarshall_exec(raw_section: &BpfRawSection, event: &mut OvsEvent)
                     21 => OvsAction::CheckPktLen,
                     22 => OvsAction::AddMpls,
                     23 => OvsAction::DecTtl,
+                    // The private OVS_ACTION_ATTR_SET_TO_MASKED action is used
+                    // in the same way as OVS_ACTION_ATTR_SET_MASKED. Use only
+                    // one action to avoid confusion
+                    25 => OvsAction::SetMasked,
```

Comment:
That's the thing, `OVS_ACTION_ATTR_SET_TO_MASKED` is kernel-only, it's not set by the user. All `OVS_ACTION_ATTR_SET_MASKED` are internally translated to  `OVS_ACTION_ATTR_SET_TO_MASKED`  when flows are validated: https://github.com/torvalds/linux/blob/ad2fd53a7870a395b8564697bef6c329d017c6c9/net/openvswitch/flow_netlink.c#L2858.

Reply:
When kernel sends flows to userspace (e.g: for printing), they are translated back to  `OVS_ACTION_ATTR_SET_TO_MASKED` or ` OVS_ACTION_ATTR_SET` (the latter is only used for tunnels). And when OVS prints them, it uses "set" for both. Not sure if we can _avoid_ confusion xD

Reply:
I don't think we need we need both values because the kernel tracepoint will never see the user-provided value and viceversa.  I would be  confused if I see two actions that are almost the same